tests/coreclr/p1/Makefile

```diff
@@ -0,0 +1,121 @@
+	JIT/Regression/JitBlue/DevDiv_461649/DevDiv_461649/DevDiv_461649.dll
+	sed -i '19d' config_1g.json
+	sed -i '19 i \ \ \ \ "CurrentWorkingDirectory":"/coreclr-tests-all/profiler/gc/gc/",' config_2g.json
+	$(RUN_SINGLE_TEST) $(MYST_EXEC) config_2g.json 900 ext2 \
+	profiler/gc/gc/gc.dll
+	sed -i '19d' config_2g.json
+	sed -i '19 i \ \ \ \ "CurrentWorkingDirectory":"/coreclr-tests-all/profiler/gc/gcbasic/",' config_2g.json
+	$(RUN_SINGLE_TEST) $(MYST_EXEC) config_2g.json 900 ext2 \
+	profiler/gc/gcbasic/gcbasic.dll
+	sed -i '19d' config_2g.json
+	sed -i '19 i \ \ \ \ "CurrentWorkingDirectory":"/coreclr-tests-all/profiler/rejit/rejit/",' config_2g.json
+	$(RUN_SINGLE_TEST) $(MYST_EXEC) config_2g.json 900 ext2 \
+	profiler/rejit/rejit/rejit.dll
+	sed -i '19d' config_2g.json
+	sed -i '19 i \ \ \ \ "CurrentWorkingDirectory":"/coreclr-tests-all/profiler/unittest/metadatagetdispenser/",' config_2g.json
+	$(RUN_SINGLE_TEST) $(MYST_EXEC) config_2g.json 900 ext2 \
+	profiler/unittest/metadatagetdispenser/metadatagetdispenser.dll
+	sed -i '19d' config_2g.json
+	sed -i '19 i \ \ \ \ "CurrentWorkingDirectory":"/coreclr-tests-all/profiler/eventpipe/eventpipe_readevents/",' config_2g.json
+	$(RUN_SINGLE_TEST) $(MYST_EXEC) config_2g.json 900 ext2 \
+	profiler/eventpipe/eventpipe_readevents/eventpipe_readevents.dll
+	sed -i '19d' config_2g.json
+	sed -i '19 i \ \ \ \ "CurrentWorkingDirectory":"/coreclr-tests-all/profiler/elt/slowpatheltenter/",' config_2g.json
+	$(RUN_SINGLE_TEST) $(MYST_EXEC) config_2g.json 900 ext2 \
+	profiler/elt/slowpatheltenter/slowpatheltenter.dll
+	sed -i '19d' config_2g.json
+	sed -i '19 i \ \ \ \ "CurrentWorkingDirectory":"/coreclr-tests-all/profiler/elt/slowpatheltleave/",' config_2g.json
+	$(RUN_SINGLE_TEST) $(MYST_EXEC) config_2g.json 900 ext2 \
+	profiler/elt/slowpatheltleave/slowpatheltleave.dll
+	sed -i '19d' config_2g.json
+
+run:
+	$(TEST_RUNNER) $(PACKAGE_PATH) null null 120 package pr1-256m 100
+	$(TEST_RUNNER) $(MYST_EXEC) config_1g.json 300 ext2 pr1-1g-passed 100 4
+	$(TEST_RUNNER) $(MYST_EXEC) config_3g.json 120 ext2 pr1-3g-passed 100 4
+	$(TEST_RUNNER) $(MYST_EXEC) config_3g.json 900 ext2  pr1-3g-900 100 4
+	$(MAKE) custom-test-runs
+	# Model number 106 or 108 indicates IceLake machines
+	if cat /proc/cpuinfo | grep -m 1 "model.*:" | grep '106\|108' > /dev/null; then \
+	$(TEST_RUNNER) $(MYST_EXEC) config_1g.json 300 ext2 pr1-icelake-1g-passed 100 4; \
+	$(TEST_RUNNER) $(MYST_EXEC) config_8g.json 2400 ext2 pr1-icelake-8g-passed 100 1; \
+	fi
+	$(STAT)
+
+package.pem:
+	openssl genrsa -out package.pem -3 3072
+
+clean:
+	sudo rm -fr appdir ext2fs rootfs roothash myst /tmp/myst* PASSED FAILED-* package.pem
+
+tests:
+	@ $(MAKE) -s clean
+	@ $(MAKE) -s all
+	@ $(MAKE) -s run
+
+##################################
+#         dev targets            #
+##################################
+
+one:
+	$(MYST_EXEC) ext2fs $(OPTS) \
+	/coreclr-tests-all/Tests/Core_Root/corerun \
+	/coreclr-tests-all/$(TEST_CASE)
```

Comment:
Maybe change to `TEST` to match ltp and libcxx test suites?

Reply:
Done